homeassistant/components/sabnzbd/__init__.py

```diff
@@ -8,39 +8,25 @@
 
 import voluptuous as vol
 
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry, ConfigEntryState
-from homeassistant.const import (
-    CONF_API_KEY,
-    CONF_HOST,
-    CONF_NAME,
-    CONF_PORT,
-    CONF_SENSORS,
-    CONF_SSL,
-    Platform,
-)
+from homeassistant.config_entries import ConfigEntry, ConfigEntryState
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant, ServiceCall, callback
 from homeassistant.exceptions import ConfigEntryNotReady, HomeAssistantError
 from homeassistant.helpers import config_validation as cv, device_registry as dr
 from homeassistant.helpers.entity_registry import RegistryEntry, async_migrate_entries
 import homeassistant.helpers.issue_registry as ir
-from homeassistant.helpers.typing import ConfigType
 
 from .const import (
     ATTR_API_KEY,
     ATTR_SPEED,
-    DEFAULT_HOST,
-    DEFAULT_NAME,
-    DEFAULT_PORT,
     DEFAULT_SPEED_LIMIT,
-    DEFAULT_SSL,
     DOMAIN,
     SERVICE_PAUSE,
     SERVICE_RESUME,
     SERVICE_SET_SPEED,
 )
 from .coordinator import SabnzbdUpdateCoordinator
 from .sab import get_client
-from .sensor import OLD_SENSOR_KEYS
```

Comment:
The `OLD_SENSOR_KEYS` in `sensor.py` is used by a different test and migration, will be removed in a second PR.